sdk/ml/azure-ai-ml/azure/ai/ml/entities/_inputs_outputs/input.py

```diff
@@ -214,9 +214,9 @@ def __init__(
         **kwargs,
     ):
         super(Input, self).__init__(type=type)
-        # As an annotation, it is not allowed to initialize the _arg_name.
-        # The _arg_name will be updated by the annotated variable _arg_name.
-        self._arg_name = None
+        # As an annotation, it is not allowed to initialize the port_name.
+        # The port_name will be updated by the annotated variable port_name.
+        self.port_name = None
```

Comment:
Can we add this as private attribute, e.g. _port_name

Reply:
replaced